lib/components/app/call-taker-panel.js

```diff
@@ -153,10 +153,12 @@ class CallTakerPanel extends Component {
     }
   }
 
+  // eslint-disable-next-line complexity
```

Comment:
I've seen this warning in other places. Should be disable those as well?

Reply:
No! It's a good reminder to make our code better